k8s/clusters/eks/modules/eks/kubectl.tf

```diff
@@ -0,0 +1,36 @@
+data "template_file" "node_drainer_configmap" {
+  template = "${file("${path.module}/templates/node-drainer-configmap.yaml")}"
+
+  vars = {
+    region               = "${var.region}"
+    lifecycled_sns_topic = "${module.lifecycle.sns_topic_arn}"
+    lifecycled_log_group = "${var.lifecycled_log_group}-${var.cluster_name}"
+  }
+}
+
+resource "null_resource" "node_drainer" {
+  depends_on = ["module.lifecycle"]
+
+  provisioner "local-exec" {
+    working_dir = "${path.module}"
+
+    command = <<EOF
+for i in `seq 1 10`; do \
```

Comment:
@limed why are you looping here?

Reply:
This is for a retry, it will retry 10 times and then quit